inst/extdata/test_phenotype.tsv

```diff
@@ -0,0 +1,7470 @@
+strain	trait1	trait2	trait3	trait4
```

Comment:
I know we talked about this format. It's highly redundant. Not a big deal but not my fav either.

Reply:
why is this redundant?